kt/godot-library/src/main/kotlin/godot/core/math/Basis.kt

```diff
@@ -677,6 +677,19 @@ class Basis() : CoreType {
         this._z = ret._z
     }
 
+    /**
+     * Just as before, there's no need for the param inside square brackets.
+     * Use the parameter names directly. Also, keep in mind that the description inside the brackets should match the actual parameter names used in the function signature.
```

Comment:
This java doc is odd. even if it's copied from godot. TBH I'd change that.

Reply:
My bad a copied pasted the wrong line